packages/components/atoms/f-button/stories/Button.stories.js

```diff
@@ -56,28 +56,32 @@ ButtonComponent.argTypes = {
 ButtonComponent.args = {
     buttonLists: [
         [
-            { type: 'primary', size: 'large', text: 'Large Primary Button' },
-            { type: 'primary', size: 'medium', text: 'Medium Primary Button' },
-            { type: 'primary', size: 'small', text: 'Small Primary Button' },
-            { type: 'primary', size: 'xsmall', text: 'Xsmall Primary Button' }
+            { type: 'primary', size: 'large', text: 'Large Primary' },
```

Comment:
Got rid of the superfluous word "Button" on each of these to free up a bit of space.